asset/asset.go

```diff
@@ -543,7 +543,13 @@ func (r *RawKeyGenesisSigner) SignGenesis(keyDesc keychain.KeyDescriptor,
 		return nil, nil, err
 	}
 
-	return tweakedPrivKey.PubKey(), sig, nil
+	// Since we'll never query lnd for a tweaked key, it doesn't matter if
+	// we lose the parity information here.
+	tweakedPubKey, _ := schnorr.ParsePubKey(
```

Comment:
I don't think we need this anymore after https://github.com/lightninglabs/taro/pull/187 ?